### PR TITLE
Add issues templates

### PR DESCRIPTION
**Pull request by diegonehab**
Copied commit from the machine-emulator repo.

### CODE REVIEW

**Review by mpolitzer — APPROVED** [review]
LGTM

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Although this software is open source and we welcome contributions,
+we believe these contributions should be preceded by an open discussion.
+Open discussions tend to result in better solutions to any given problem,
+and help maintain and improve the quality of the software.
+If you would like to see a bug fixed or a new feature implemented,
+please open an issue for the discussion rather than directly opening
+a pull request.
+
+If you would like to contribute please read [CONTRIBUTING.md](https://github.com/cartesi/machine-emulator/blob/main/CONTRIBUTING.md).
```
Comment (mpolitzer) [inline .github/pull_request_template.md]: Can this be repo agnostic?

**Review by mpolitzer — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (diegonehab) [reply .github/pull_request_template.md]: Done. Made it relative.

**Review by mpolitzer — DISMISSED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Although this software is open source and we welcome contributions,
+we believe these contributions should be preceded by an open discussion.
+Open discussions tend to result in better solutions to any given problem,
+and help maintain and improve the quality of the software.
+If you would like to see a bug fixed or a new feature implemented,
+please open an issue for the discussion rather than directly opening
+a pull request.
+
+If you would like to contribute please read [CONTRIBUTING.md](../../blob/main/CONTRIBUTING.md).
```
Comment (edubart) [inline .github/pull_request_template.md]: I think @vfusco wants this to be changed like he did in other repositories.

**Review by edubart — APPROVED** [review]

**Review by vfusco — APPROVED** [review]

**Review by mpolitzer — APPROVED** [review]

**Pull request merged**